Kudu.Contracts/Deployment/ArtifactType.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Kudu.Contracts.Deployment
+{
+    public enum ArtifactType
+    {
+        Invalid,

```

Comment:
What does the Invalid ArtifactType represent ? Is it that when we can't recognize the artifact type from the query param ? If yes, could we rename it to `Unknown`

Reply:
Done